controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -482,18 +489,25 @@ bool master_thread::handle_cmdu_1905_autoconfiguration_WSC(Socket *sd,
         return false;
     }
 
+    // trigger channel selection
```

Comment:
As discussed, please add a parameter in the controller config - `certification_enabled`
If `certification_enabled`, don't automatically trigger channel selection.

Reply:
Tomer, I thought about it and I prefer to wait with this. Maybe it will not be necessary. At most, when I will create a WFCA task, I will check if the WFCA task exists and will know not to send it.